### ISSUE TITLE
Quasar Infastructure: Migrate Looker Server

### ISSUE THREAD

**Issue by jessleenyc** [DoSomething/quasar#229]
_TASK_
-------------------
#### Request
Migrate Looker Server for Quasar Infrastructure. 

#### Additional Information
Original issue here: https://github.com/DoSomething/quasar/issues/185

#### Why This Matters/Use Case Example
Foundational task to getting Quasar up and running!

**Comment by jessleenyc** [comment]
@sheyd we're already migrating the looker server?! I thought that would be one of the last steps, can you tell me more?

**Comment by sheyd** [comment]
Pending Morgan's approval of PR.

**Comment by sheyd** [comment]
PR approved, and server setup with rest of tunnel config. Now just time to pull the big red lever...

**Comment by sheyd** [comment]
The HA Proxy config for Looker was giving a 502 incomplete gateway error. At first I suspected proxy setting issues, so installed Nginx as an HTTP proxy on the Looker, but still got the same error. Looker uses HTTPS on port 9999 by default, but the cert is self-signed, so I had to update the HA Proxy config to have `ssl verify none` on the `server` line so traffic passed properly. Waiting to pull big red-lever a little after lunchtime.
